iOS/Controllers/CoursesViewController.swift

```diff
@@ -23,6 +23,8 @@ class CoursesViewController: UICollectionViewController, UICollectionViewDelegat
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        NotificationCenter.default.addObserver(self, selector: #selector(contentSizeDidChanged), name: UIContentSizeCategory.didChangeNotification, object: nil)
+
```

Comment:
You can use `override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?)` to observe the changes to the preferred content size. No need to listen to a notification.